packages/key-readmodel-writer/test/consumerService.integration.test.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable functional/immutable-data */
+/* eslint-disable functional/no-let */
+import { afterEach, afterAll, beforeAll, describe, expect, it } from "vitest";
+import {
+  KeyCollection,
+  ReadModelRepository,
+  readModelWriterConfig,
+} from "pagopa-interop-commons";
+import { mongoDBContainer } from "pagopa-interop-commons-test";
+import { StartedTestContainer } from "testcontainers";
+
+describe("Integration tests", async () => {
+  let keys: KeyCollection;
+  let startedMongoDBContainer: StartedTestContainer;
+
+  const config = readModelWriterConfig();
+
+  beforeAll(async () => {
+    startedMongoDBContainer = await mongoDBContainer(config).start();
+
+    config.readModelDbPort = startedMongoDBContainer.getMappedPort(27017);
+
+    const readModelRepository = ReadModelRepository.init(config);
+    keys = readModelRepository.keys;
+  });
+
+  afterEach(async () => {
+    await keys.deleteMany({});
+  });
```

Comment:
This is the old way to initialize test containers, can you update it?

Reply:
Updated, thanks